stumpy/scrump.py

```diff
@@ -151,52 +162,60 @@ def _compute_PI(
                     - T_B[i + g - 1] * T_A[j + g - 1]
                     + T_B[i + g + m - 1] * T_A[j + g + m - 1]
                 )
-                D_squared = core._calculate_squared_distance(
+                d_squared = core._calculate_squared_distance(
```

Comment:
Maybe switch back to `D_squared`?

Reply:
Yes, I think it is better to keep it as `D_squared`. I don't remember exactly but I may have been trying to keep the use of `D` consistent with the original `stomp`/`stamp` papers to refer to individual distances.